charts/azuremonitor-containers/templates/omsagent-rs-configmap.yaml

```diff
@@ -108,24 +108,24 @@ data:
      </match>
 
      <match oms.containerinsights.KubePVInventory**>
-     type out_oms
-     log_level debug
-     num_threads 5
-     buffer_chunk_limit 4m
-     buffer_type file
-     buffer_path %STATE_DIR_WS%/state/out_oms_kubepv*.buffer
-     buffer_queue_limit 20
-     buffer_queue_full_action drop_oldest_chunk
-     flush_interval 20s
-     retry_limit 10
-     retry_wait 5s
-     max_retry_wait 5m
+      type out_oms
+      log_level debug
+      num_threads 5
```

Comment:
Does this also need to be changed to 'num_threads 2' like KubePodInventory, KubeEvents etc. ? I also have it as 5 in omsagent.yaml

Reply:
> PV is not high volume data so I think we are fine with num_threads as 5 for PV.  If the data type is high volume then higher number of threads will make size of the file reads are large which will cause OOM kill if there is not enough memory. The optimizations which I have done are adjusting number of threads to avoid large file reads which in turn handle the OOM killer situtation.